src/main/java/teammates/ui/output/UsageStatisticsRangeData.java

```diff
@@ -1,19 +1,28 @@
 package teammates.ui.output;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import teammates.common.datatransfer.attributes.UsageStatisticsAttributes;
+import teammates.storage.sqlentity.UsageStatistics;
 
 /**
- * The API output format of a list of {@link UsageStatisticsAttributes}.
+ * The API output format of a list of {@link UsageStatisticsAttributes} and {@link UsageStatistics}.
  */
 public class UsageStatisticsRangeData extends ApiOutput {
 
     private final List<UsageStatisticsData> result;
 
-    public UsageStatisticsRangeData(List<UsageStatisticsAttributes> usageStatistics) {
-        this.result = usageStatistics.stream().map(UsageStatisticsData::new).collect(Collectors.toList());
+    public UsageStatisticsRangeData(
+            List<UsageStatisticsAttributes> usageStatistics,
+            List<UsageStatistics> sqlUsageStatistics) {
+        List<UsageStatisticsData> allData = new ArrayList<>();
+
+        allData.addAll(usageStatistics.stream().map(UsageStatisticsData::new).collect(Collectors.toList()));
+        allData.addAll(sqlUsageStatistics.stream().map(UsageStatisticsData::new).collect(Collectors.toList()));
+
```

Comment:
No need to support old schema for this entity. This will be migrated as one of the first entities as it can be done independently from the rest of the entities and with no real disruption to regular users. Not worth the hassle to support both old and new entities at the moment.